.github/workflows/scheduled.yaml

```diff
@@ -8,9 +8,13 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  property-testing-v1:
-    name: Clarity::V1 Property Tests
+  property-testing:
     runs-on: ubuntu-latest
+    if: ${{ always() }}
```

Comment:
qq: why do we need `if: ${{ always() }}` here?

Reply:
To keep running other jobs even if a fail happens.

Reply:
But here we do not have other jobs (other than the strategy matrix); if it's related to the strategy, shouldn't it be [`fail-fast: false`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#handling-failures)?

Reply:
Oh, yep... its only for the strategy. 👍 Changed.